mods/stairs/init.lua

```diff
@@ -256,355 +373,425 @@ if replace then
 end
 
 
--- Stair/slab registration function.
--- Nodes will be called stairs:{stair,slab}_<subname>
+-- Stair/slab/stairi/stairo registration function.
+-- Nodes will be called stairs:{stair,slab,stairi,stairo}_<subname>
 
-function stairs.register_stair_and_slab(subname, recipeitem,
```

Comment:
I just used tenplus1's code from Stairs Redo (with very slight modifications) for backward compat, tested with quartz mod and castles mod... works fine now.